### PR TITLE
update README info and update host network to net mode

### PR DESCRIPTION
**Pull request by Muzry**
- update README info and update host network to net mode
- @luobily

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,60 @@
-# erda-release
+# Erda Installation Tools
```
Comment (luobily) [inline README.md:1]:
```suggestion
# Erda Release Tools
```

File: README.md [hunk 2]
```diff
@@ -1,46 +1,60 @@
...
 
-The erda release installer  pass the test on following software version:
+We have successfully installed Erda in the following software:
 
 - Kubernetes 1.18.8
```
Comment (luobily) [inline README.md:5]:
```suggestion
- Kubernetes 1.18+
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,60 @@
...
 - Docker 19.03.5
 - CentOS 7.9
+- Helm 3.4.2
```
Comment (luobily) [inline README.md:8]: These all require a version range

File: README.md [hunk 4]
```diff
@@ -1,46 +1,60 @@
...
 
 - Kuberentes 1.18.8, We recommand you use ACK Pro ( AliCloud Kubernetes Pro)
-  - The minimal node resource is 4 CPU 16G Memory
-  - At least 4 Node (1 Master and 3 Worker)
-  - 200G Storage on `/`
-  - **Don't Install the ingress**
+  - Each node needs at least 4 core CPU, 16G memory
+  - At least 4 Nodes (1 Master and 3 Workers)
+  - At least 200G storage in the `/`
+  - Need an SLB
```
Comment (luobily) [inline README.md:20]: delete "We recommand you use ACK Pro ( AliCloud Kubernetes Pro)"

Comment (luobily) [reply README.md:20]: delete "- Need an SLB"

File: README.md [hunk 5]
```diff
@@ -1,46 +1,60 @@
...
+  - **Don't Install the ingress component**
 - Docker 19.03.5
-- CentOS 7
-- Helm 3
+- CentOS 7 +
+- Helm 3.4 +
```
Comment (luobily) [inline README.md:24]: The whole process should not be related to Alibaba Cloud，e.g. ACK、SLB

File: README.md [hunk 6]
```diff
@@ -1,46 +1,60 @@
...
+
+
 
 ### Step
```
Comment (luobily) [inline README.md:28]: Divided into two steps：
1. release package;
2. erda install

File: README.md [hunk 7]
```diff
@@ -102,17 +143,21 @@ The erda release installer  pass the test on following software version:
    <SLB_IP> hepa.erda-demo.erda.io
    <SLB_IP> openapi.erda-demo.erda.io
    <SLB_IP> uc.erda-demo.erda.io
-   <SLB_IP> erda-test-org.erda-demo.erda.io
+   <SLB_IP> <orgname>-org.erda-demo.erda.io
    <SLB_IP> test-java.erda-demo.erda.io
```
Comment (luobily) [inline README.md:147]: SLB_IP -> IP

**Review by luobily — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (Muzry) [reply README.md:147]: Done

File: README.md (hunk 6 above)
Comment (Muzry) [reply README.md:28]: Done

File: README.md (hunk 5 above)
Comment (Muzry) [reply README.md:24]: Done

File: README.md (hunk 3 above)
Comment (Muzry) [reply README.md:8]: There are the software versions that successfully installed Erda，I think we should inform users clearly.

File: README.md [hunk 8]
```diff
@@ -1,46 +1,63 @@
...
 
-4. copy the tar ball to your  **Kubernetes Master** node and make sure the **kubeconfig** file on the ~/.kube/config, then prepare the following enviroment variables on the **Kubernetes Master Node**.
 
-   >  scp package/erda-release.tar.gz root@<hostip>:/root
+
+#### Erda Install
+
+1. Copy the tarball to your  **Kubernetes Master** node and make sure the **kubeconfig** file on the ~/.kube/config.
+
+   > scp package/erda-release.tar.gz root@<hostip>:/root
```
[the remark this answers is not in the document]
Comment (Muzry) [reply README.md]: Done

**Review by luobily — APPROVED** [review]
lgtm

**Pull request merged**